### ISSUE TITLE
Fensterbretter im hauptraum

### ISSUE THREAD

**Issue by Bfritz0815** [raumzeitlabor/rzl-tuwat#172]
Hallo,
Winter is coming. 
Um den Raum etwas besser warm halten zu können, hätte ich gerne Fensterbretter im Hauptraum. 
Das sollte relativ wenig kosten und sowohl für die Heizerei  als auch die Ästhetik was bringen.

Meinungen?

**Comment by steeph-k** [comment]
Dank sortiertem Holzlager sollten sich ja jetzt schenll passende Bretter 
finden.

**Comment by Don42** [comment]
Bei der Tiefe der Sockel ist das glaube ich eher unwahrscheinlich. In dem Format hatte ich glaube ich kein Holz gesehen.
Wie viel Platz haben wir denn unter den geöffneten Fenstern?

**Comment by steeph-k** [comment]
On 18.11.2017 20:39, Marco Kaulea wrote:
>
> Wie viel Platz haben wir denn unter den geöffneten Fenstern?
>
Ca. 60 mm

Wir könntenauch die Arbeitsplatten aus der alten Küche dafür verwenden. 
Und für den Rest Arbeitsplattenreste aus dem Baumarkt.

**Comment by Don42** [comment]
Ok, bei 60mm haben wir ja Spielraum.
Die Arbeitsplatten die wir da haben zu verwenden finde ich grundsätzlich sinnvoll,
aber extra Arbeitsplatten kaufen macht keinen Sinn. Wie sieht das den mit Resten aus?
Was kosten die?  Sind Arbeitsplatten nicht schon ~5cm hoch?

**Comment by steeph-k** [comment]
On 18.11.2017 20:52, Marco Kaulea wrote:
>
> Wie sieht das den mit Resten aus?
> Was kosten die?
>
Im Baumarkt gar nicht mal so viel weniger als die Meterware. Bei ebay 
Kleinanzeigen oftmals nichts.

> Sind Arbeitsplatten nicht schon ~5cm hoch?
>
Es gibt Arbeitsplatten, die dünner als 2 cm und welche, die dicker als 8 
cm sind. Über 5 cm würde ich nicht gehen, damit auf jeden Fall noch 
etwas Abstand zu den Fenstern bleibt. Die jetzigen Platten sind nämlich 
uneben und ich würde mich nicht darauf verlassen, dass der Abstand auf 
der ganzen Breite genau gleich ist.

**Comment by Don42** [comment]
Wenn wir unsere Alten wieder verwenden wollen, müssten die die wir dazu kaufen genau so hoch sein.
Oder wir nehmen für alles neues Holz.

**Comment by uwap** [comment]
Wenn wir Holzbretter für die Fenster holen müssen wir auf jeden Fall daran denken, dass die eine kleine Metallschiene haben, damit die Magneten der Gardinen da weiter dran haften können.

**Comment by Bfritz0815** [comment]
Metallschine machen wir nachträglich drauf

**Comment by steeph-k** [comment]
![imag1053](https://user-images.githubusercontent.com/3319490/35212933-2a1e2a2a-ff5c-11e7-9e08-f87d5bfd0400.jpg)
![imag1057](https://user-images.githubusercontent.com/3319490/35212934-2a39b416-ff5c-11e7-80a3-b04694a8228d.jpg)
Die wären in Bad Dürkheim abzuholen. Von letzteren gibt es noch mehr. Die müssten aber noch lackiert werden oder so.

**Comment by Bfritz0815** [comment]
möche gerne noch vor dem Anschrauben der Bretter den Hohlraum drunter ausschäumen. 
Die Bretter halten zwar die Konvektion auf, könnten aber ohne Ausschäumung schneller selbst auskühlen. 

Meinungen?

**Comment by tabascoeye** [comment]
Ja. Bauschaum dehnt sich aus und die Konstruktion unter den Fenstern ist Rigips auf Metallständern. Das wird im Zweifel einfach kaputt brechen unter Druck.
Ich hab wenig Lust das dem Vermieter zu erklären.

**Comment by uwap** [comment]
Maße: 3000 Breite, 300 Tiefe, 28 Höhe.
Benötigen wir 4 mal. Gussmamor wäre das beste. Abdichten könnten wir das mit Acryl.

=> Wir sollten klären ob Mutlu das vielleicht finanzieren müsste

**Comment by Bfritz0815** [comment]
Habe heute etwas eingekauft:

Acryl für die fugen oben an den Fensterbrettern

Rohr für die ätzecke

Schnellzement 10 kg für unterhalb der Fensterbretter


Bringe das Zeug die Tage in Richtung Raum

uwap <notifications@github.com> schrieb am So., 2. Sep. 2018, 16:53:

**Comment by uwap** [comment]
Die Frage ob unser Vermieter nicht vielleicht für die Fensterbretter zuständig ist bleibt trotzdem offen. Vielleicht haben @tabascoeye oder @flederrattie da einen besseren überblick?

**Comment by s1lvester** [comment]
Ist doch egal ob der zuständig ist. Selbst wenn passiert die nächsten 2 Jahre eh nichts (Vgl. Nebenkostenabrechnung). Kosten sind Überschaubar und @Bfritz0815 scheint versiert und motiviert. Warum jetzt kaputt diskutieren?

**Comment by Bfritz0815** [comment]
@uwap mir gehts da nicht um eine Durchführung oder Finanzierung. (Beides können wir bei Mutlu wahrscheinlich eh vergessen). 

Wichtiger ist mir die Genehmigung für etwas, das die Räumlichkeiten verändert. 

Wenn wir da schon @tabascoeye und/oder @flederrattie involvieren, dann lieber in Form von: "Bitte schaut mal in den Mietvertrag, ob und inwieweit wir selbst Renovierungen durchführen können (oder sogar müssen?)"

Der Zement hat ca 10€ gekostet, ads Acryl auch etwa in der Richtung. Die Fensterbretter werden auch nicht in den 4 stelligen Bereich gehen. Die Bretter selbst aus Gussmarmor werden incl Versand wahrscheinlich die 100€ knapp durchstoßen. 

Damit bliebe das Projekt im Ganzen unter 200€ und könnte per Plenum abgestimmt werden. 

Ich finde nur:
Bis zu 200€ in die hand nehmen, Zeit und Mühe investieren und dann rausfinden, dass wir das wieder rausreißen müssen, wäre wie die Briten sagen: "rather dämlich".
